Updated README with links to relevant issues. (#345)
GoogleCloudPlatform/oozie-to-airflow: README.md

> support for all EL functions. So in order for it to run in Airflow you must

Reviewer: ```suggestion
 support for all EL functions. So in order for it to run in Airflow you may need to
```

I think that decision node implemented in #326 should handle most of the simple predicates.

Currently the only supported way of configuring the map-reduce action is with the
inline action configuration, i.e. using the `<configuration>` tag in the workflow's XML file definition.

Issues in Github:
* [Add support for config-default.xml](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/137)
* [Add support for parameters section of the workflow.xml](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/138)
* [Handle global configuration properties](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/134)

**3. Streaming and pipes**

Streaming and pipes are currently not supported.

> Issue in github [Implement streaming support](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/336)

Reviewer 2: ```suggestion

Issue in github [Implement streaming support](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/336)
```
This will start the sentence in a new line.

### Known limitations

Not all FS operations are currently idempotent. It's not a problem if prepare action is used in other tasks

> but might be a problem in certain situations. Fixing the operators to be idempotent requires more complex

Reviewer: As a new o2a user I would be glad to know an example of a "certain situation" 🙂

Reviewer 3: Duplicate: https://github.com/GoogleCloudPlatform/oozie-to-airflow/pull/345/files#r303486063

> but might be a problem in certain situations. Fixing the operators to be idempotent requires more complex

Reviewer: As a new o2a user I would be really helpful to know an example of a "certain situation" 🙂

Reviewer 3: Your request required for a deeper analysis. Unfortunately, we did not do it because we were under time pressure.

Reviewer: Ok, I understand

### Known limitations

1. Overriding action's Main class via `oozie.launcher.action.main.class` is not implemented.

> Issue in Github: [Override Java main class with property](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/338)

Reviewer 2: ```suggestion

Issue in Github: [Override Java main class with property](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/338)
```
